bigquery/samples/list_datasets_projects.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Command-line skeleton application for BigQuery API.
+
+Usage:
+  $ python list_datasets_projects.py
+"""
+
+import pprint
+import sys
+from urllib2 import HTTPError
+
+from apiclient import discovery
+
+from oauth2client.client import GoogleCredentials
+
+
+# [START listDatasets]
+def ListDatasets(service, project):
+    try:
+        datasets = service.datasets()
+        list_reply = datasets.list(projectId=project).execute()
+        print 'Dataset list:'
+        pprint.pprint(list_reply)
+
+    except HTTPError as err:
+        print 'Error in ListDatasets:', pprint.pprint(err.content)
+# [END listDatasets]
+
+
+# [START listProjects]
+def ListProjects(service):
+    try:
+        # Start training on a data set
+        projects = service.projects()
+        list_reply = projects.list().execute()
+
+        print 'Project list:'
+        pprint.pprint(list_reply)
+
+    except HTTPError as err:
+        print 'Error in ListProjects:', pprint.pprint(err.content)
+# [END listProjects]
+
+
+def main(argv):
```

Comment:
This is quick and dirty, but I would prefer you to use argparse with one argument.